operator/v1/types_network.go

```diff
@@ -328,6 +331,19 @@ type HybridOverlayConfig struct {
 	HybridOverlayVXLANPort *uint32 `json:"hybridOverlayVXLANPort,omitempty"`
 }
 
+type OVSHardwareOffloadConfig struct {
+	// enableHardwareOffload specifies whether or not OVS hardware offload
+	// should be enabled. If unset, this property defaults to 'false' and
+	// OVS hardware offload is disabled.
+	EnableHardwareOffload bool `json:"enableHardwareOffload"`
+	// hardwareOffloadPolicy is the Traffic Control Policy (tc-policy) specified
+	// for OVS instance. This field is only relevant if EnableHardwareOffload is
+	// enabled. If unset, this property defaults to TrafficControlPolicyNone. The
```

Comment:
If it's allowed for it to be unset then add `// +optional`

Reply:
`EnableHardwareOffload is required and `HardwareOffloadPolicy` is optional, I will mark the latter as optional.

Reply:
Done